sisl/io/sile.py

```diff
@@ -318,6 +319,37 @@ def get_siles(attrs=None):
     return siles
 
 
+@set_module("sisl.io")
+def get_sile_rules(attrs=None, cls=None):
+    '''
+    Retrieve all sile rules of siles with specific attributes or methods
+
+    Parameters
+    ----------
+    attrs : list of attribute names
+       limits the returned objects to those that have
+       the given attributes ``hasattr(sile, attrs)``, default ``[None]``
+    '''
+    global __sile_rules
+
+    if attrs is None and cls is None:
+        return list(__sile_rules)
+
+    sile_rules = []
+    for rule in __sile_rules:
+        sile = rule.cls
+        if sile is cls:
+            sile_rules.append(rule)
+            continue
```

Comment:
this is superfluous, no? delete `continue`

Reply:
It is indeed :)